rhpam-operator.yaml

```diff
@@ -2474,7 +2474,7 @@ spec:
         - name: RUNTIME_IMAGE
           value: rhpam-kogito-runtime-jvm-rhel8
         - name: IMAGE_REGISTRY
-          value: registry.stage.redhat.io/rhpam-7
+          value: registry.redhat.io/rhpam-7
```

Comment:
@radtriste @AntStephenson @rgdoliveira @jakubschwan 
do you guys think we can use the official registry directly or, is it mandatory to use the stage here?

Reply:
we can replace it with scripts with the make bundle command.
We tried to rely on osbs but it didn't work as expected.

Reply:
discussed on gchat.